heatpump.yaml

```diff
@@ -3189,23 +3078,49 @@ number:
       value_bytes[0] = value >> 8;     // high byte (zone 2)
       value_bytes[1] = value & 0x00FF; // low byte (zone 1)
 
-      // ESP_LOGI("","Zone 1 is %d", value_bytes[1]);
-      // ESP_LOGI("","Zone 2 is %d", value_bytes[0]);
+      auto operation_state = id(${devicename}_operational_mode).state;
```

Comment:
Can you explain what this code exactly does. I seems to me that the value (read and write) is changed based on the emission type.

We should not interfere with the input of the user, if the user specifies a certain value, then that value should be used. We must only limit the value of the slider, based on the limits of the specification in the manual

Reply:
Rereading your code helped 😉 What you do is set the limit based on the emission type. I see two issues with this: 

1) The user sets a value, but the system changes that value without feedback to user. If the user does not verify, another value can be used then he/she wanted.
2) The limits are hard coded and are easily forgotten if these are changed somewhere else.

I rather keep it simple and only define the upper and lower limit, thus as it already is.